content/en/docs/examples/local-testing-with-kind/index.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Testing Istio Locally With Kubernetes In Docker
+description: Example of deploying Istio locally using Kubernetes In Docker for testing and evaluation.
+weight: 10
+keywords:
+- kubernetes
+- kind
+- metallb
+- cert-manager
+- ingress
+test: true
```

Comment:
Remove this line, unless you are planning to write a test program for the page: https://github.com/istio/istio.io/tree/master/tests#testing-istioio-content.
If you remove this field, you can also remove the generated snip.sh file.

Reply:
Before you delete snip.sh, you might want to take a look at the reported lint errors, to see if they are legitimate bad things in your code blocks or not.
```
If there are any lint errors in the generated snip.sh file, it means that a command in the
index.md file is not following bash best practices. Because we are extracting the commands
from the markdown file into a script file, we get the added benefit of lint checking of the
commands that appear in the docs.
```

Reply:
Looking at the lint error in the snip.sh, the problem is that you are using `<< EOF` in some of the commands. The doc framework doesn't recognize this as a multi-line bash command, it only handles `<<EOF` (no whitespace). The generated snip.sh ends up being bad, but more importantly, the copy button on the generated web page will also not work properly. Please change them to `<<EOF`.

Reply:
Changed, thank you :+1: